containers.go

```diff
@@ -113,7 +113,7 @@ type rwContainerStore interface {
 	// read-write layer.
 	// The maps in the container's options structure are recorded for the
 	// convenience of the caller, nothing more.
-	Create(id string, names []string, image, layer, metadata string, options *ContainerOptions) (*Container, error)
+	Create(id string, names []string, image, layer string, options *ContainerOptions) (*Container, error)
```

Comment:
In all three C/I/L stores: Now that we are touching the method at all, maybe rename it to be lowercase. That would make it clearer that this is an internal API we can change at any time.

Reply:
No objection here, changing it.